src/fmu/tools/sensitivities/design_distributions.py

```diff
@@ -9,6 +9,7 @@
 import numpy.linalg as la
 import pandas as pd
 import scipy.stats
+from scipy.stats import qmc
```

Comment:
But bear in mind that this module was not supported in scipy 1.2, which is set as minimal in `pyproject.toml`.

Reply:
Good catch. I've bumped the version.